pkgs/tools/security/pass/default.nix

```diff
@@ -134,19 +136,19 @@ stdenv.mkDerivation rec {
 
   postFixup =
     ''
-      # Fix program name in --help
-      substituteInPlace $out/bin/pass \
-        --replace 'PROGRAM="''${0##*/}"' "PROGRAM=pass"
+        # Fix program name in --help
+        substituteInPlace $out/bin/pass \
+          --replace 'PROGRAM="''${0##*/}"' "PROGRAM=pass"
```

Comment:
That extra indentation here is wrong

Reply:
This is the result of running `nixfmt-rfc-style` on `pkgs/tools/security/pass/default.nix`. Running `nixfmt-rfc-style --check ./pkgs/tools/security/pass/default.nix` outputs nothing while returning successfully.

Are you certain that the indentation should be removed?

Reply:
If people have hard feelings, a bug report may be worth it.

That said, please split formatting changes into other commits. This makes reviewing code painfully annoying.

Reply:
I've splitted the commits, then I was told to squash them:

1. https://github.com/NixOS/nixpkgs/pull/313420#pullrequestreview-2092360095
1. https://github.com/NixOS/nixpkgs/pull/313420#issuecomment-2148588865

Reply:
Oof :facepalm: 

> Formatting individual packages just creates chores, please just drop that commit.

@SuperSandro2000 you do realize that it's super-painful to review code like this?!